client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -324,6 +324,11 @@ export class MotionListComponent extends BaseListViewComponent<ViewMotion> imple
                     ],
                     fieldset: 'list',
                     additionalFields: ['text']
+                },
+                {
+                    idField: 'motion_category_ids',
+                    fieldset: 'list',
+                    follow: ['parent_id', 'child_ids']
```

Comment:
You do not have to follow them, just request them as either additional fields or create a fieldset. This is because all categories are already included in the request itself; the ids are needed by the client for the relations itself.